lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -29,8 +29,8 @@ class DevnetClient(
     private val host: String = "0.0.0.0",
     private val port: Int = 5050,
     private val httpService: HttpService = OkHttpService(),
+    private val accountDirectory: Path = Paths.get("src/test/resources/account"),
 ) : AutoCloseable {
```

Comment:
I think it's a fine initial solution. One idea for possible refinement, would be to use random UUID for file name on every instance creation, so we wouldn't have to explicitly set up a path per every test module (we could create an issue for that if you think that's useful).

Reply:
I the way we have it now makes it easier to debug some account related things later on (eg right away you see what were the addresses for providerTest)